client/my-sites/plans-v2/use-get-plans-grid-products.ts

```diff
@@ -61,33 +63,59 @@ const useSelectorPageProducts = ( siteId: number | null ) => {
 		availableProducts = [ ...availableProducts, PRODUCT_JETPACK_CRM, PRODUCT_JETPACK_CRM_MONTHLY ];
 	}
 
-	// If Jetpack Backup is directly or indirectly owned, continue, otherwise make it available by displaying
-	// the option cards.
-	if (
-		! ownedProducts.some( ( ownedProduct ) => JETPACK_BACKUP_PRODUCTS.includes( ownedProduct ) )
-	) {
-		// The Alternative Selector page include Jetpack Backup Daily rather than the option card.
-		const backupProductsToShow = [];
-		switch ( getJetpackCROActiveVersion() ) {
-			case 'v0':
-				backupProductsToShow.push( OPTIONS_JETPACK_BACKUP, OPTIONS_JETPACK_BACKUP_MONTHLY );
-				break;
-			case 'v1':
-				backupProductsToShow.push(
-					PRODUCT_JETPACK_BACKUP_DAILY,
-					PRODUCT_JETPACK_BACKUP_DAILY_MONTHLY
-				);
-				break;
-			case 'v2':
-				// TODO: add Backup Real-time
-				backupProductsToShow.push(
-					PRODUCT_JETPACK_BACKUP_DAILY,
-					PRODUCT_JETPACK_BACKUP_DAILY_MONTHLY
-				);
-				break;
+	const backupProductsToShow = [];
```

Comment:
I'm not particularly proud of what comes next. Basically, each version of the Plans grid has a different rule about what Jetpack Backup card to show depending on what the site owns. I'm leaving this comment here so we can think about this particular case and others sure will arise during v2's development.